image_cropping/widgets.py

```diff
@@ -54,6 +60,22 @@ def get_attrs(image, name):
         return {}
 
 
+_static = None
+
+def static(path):
+    if apps is None:
+        # Django < 1.7 compatibility
+        return path
+
+    global _static
```

Comment:
It would be obviously desirable if we wouldn't have to copy and duplicate the code from the [static tag](https://github.com/django/django/blob/037f81fedcbe860bacea01fa6415849e0e379c15/django/contrib/admin/templatetags/admin_static.py#L10).
I wonder if we couldn't just reuse the admin's static tag as a sort of utility function at this point?
```
from django.contrib.admin.templatetags.admin_static import static as admin_static
...
return admin_static(path)